packages/@aws-cdk/aws-servicediscovery/lib/service.ts

```diff
@@ -0,0 +1,133 @@
+import cdk = require('@aws-cdk/cdk');
+import { CfnService} from './servicediscovery.generated';
+
+export interface ServiceProps {
+
+  /**
+   * The ID of the namespace that you want to use for DNS configuration.
+   */
+  namespaceId: string;
+
+  /**
+   * The DNS type of the record that you want AWS Cloud Map to create. Supported record types
+   * include A, AAAA, and SRV.
+   */
+  dnsType: string
+
+  /**
+   * The amount of time, in seconds, that you want DNS resolvers to cache the settings for this
+   * record.
+   *
+   * @default 60
+   */
+  dnsTTL?: string
```

Comment:
Add `Sec` at the end of the field.

`dnsTtlSec`.

Let's make it a number and stringify in the constructor.